composer.json

```diff
@@ -14,7 +14,7 @@
         "ext-json": "*",
         "ext-pcntl": "*",
         "ext-posix": "*",
-        "cakephp/chronos": "^1.0",
+        "cakephp/chronos": "^1.0|^2.0",
```

Comment:
Why did you change this later on? I think the bump to just `^2.0` is perfectly fine.

Reply:
In case there are projects that use version 1

Reply:
Those projects need to define the chronos dependency in their own composer.json is they depend on it directly. We're changing the dependency for Horizon here.

Reply:
Ok, I leave it like it was before